relay/relay_test.go

```diff
@@ -443,6 +443,13 @@ func (t *testRelaySuite) TestHandleEvent(c *C) {
 	c.Assert(pos, DeepEquals, binlogPos)
 	c.Assert(gs.String(), Equals, "") // no GTID sets in event yet
 
+	// after handle rotate event, we save and flush the meta immediately
```

Comment:
if you want to test "after handle rotate event, we save and flush the meta immediately", don't start another handleEvents because previous one handled rotate event